syntax = "proto3";

package Renci;

message Response
{
    repeated string messages = 1;
}

message Post
{
    uint64 identification = 1;
    string message = 2;
}

service ChatRoom
{
    rpc sendMessage(Post) returns (Response) {}

    rpc checkForMessages(Post) returns (Response) {}
}
